config/core/test-configs.yaml

```diff
@@ -2248,6 +2248,12 @@ test_configs:
       - kselftest-futex
       - kselftest-lib
       - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ipc
+      # ltp-mm - Runs board out of memory
```

Comment:
If the memory size is a hard requirement then this will never be able to run, in which case adding a comment isn't useful.  I think we need other ways of tracking things that don't get documented well in YAML.  There's a section in the documentation about tests run by KernelCI, and there's GitHub issues for things that can be fixed.

Maybe with a better configuration system we would just add a filter to block `ltp-mm` on this platform, so the config would be both functional and self-documenting.

Reply:
I'd say this is a bug in the testsuite - it should be skipping tests that are going to run the system out of memory, if it's just exploding easily that's not really usable.

Reply:
I do also find it useful to have comments in the file explaining why something's not enabled - especially with the way we're currently mixing lab config in with board capability stuff. I know there's a few things that I've gone and looked at enabling multiple times which I'd not have done with a note present.

Reply:
OK fair enough. As discussed in the meeting today, we can use comments for the time being and think of a better way of dealing with this kind of issues in the longer term.